docs/security/multisig.md

```diff
@@ -0,0 +1,51 @@
+# Multisig
+
+## How it works
+
+Yearn's governance is managed by Governance 2.0, which was introduced with the [YIP-61 Proposal](https://gov.yearn.finance/t/yip-61-governance-2-0/10460). Through Governance 2.0, YFI holders delegate powers to the Governance Multisig.
```

Comment:
YIP-41 delegated power to the multisig only. But Gov 2.0 is about delegating powers to multiple yTeams. Check this: https://gov.yearn.finance/t/yip-61-governance-2-0/10460#yteams-9

Since this is only about the multisig, you can say that only some powers are delegated to the multisig.

Reply:
I think I'll remove this first sentence, it probably belongs in a dedicated governance document anyways